ironfish/src/account/accounts.ts

```diff
@@ -574,10 +574,10 @@ export class Accounts {
     let confirmedBalance = BigInt(0)
 
     for (const note of notes) {
-      const value = note.note.value()
-      unconfirmedBalance += BigInt(value)
+      const value = note.note.value().valueOf()
```

Comment:
I'm down in another PR, but shouldn't this method just return `bigint` to avoid this call?